README.md

```diff
@@ -10,10 +10,25 @@ their site builds. That means that a developer should never have to undo
 something that Lightning has done. So we started with a blank slate and
 justified each addition from there.
 
+## Installing Lightning
+The preferred way to install Lightning is using our
+[Composer-based project template][template]. It's easy!
+
+```
+$ composer create-project acquia/lightning-project MY_PROJECT
+```
+
+If you don't want to use Composer, you can install Lightning the traditional way
+by downloading a tarball from our
+[drupal.org project page](https://www.drupal.org/project/lightning).
+
+You can customize your installation using [lightning.extend.yml](https://github.com/acquia/lightning/blob/8.x-1.x/lightning.extend.yml). Using this file, you can choose which Lightning features to install as well as install your own custom features.
+
+## Functional areas
```

Comment:
Not sure I'd use this particular wording; "functional areas" is very technical-sounding. Maybe just "What Lightning Does"?